src/fides/api/schemas/privacy_notice.py

```diff
@@ -119,40 +130,6 @@ def validate_notice_key(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
         return values
 
-    @root_validator
-    def validate_consent_mechanisms_and_display(
-        cls, values: Dict[str, Any]
-    ) -> Dict[str, Any]:
-        """
-        Add some validation regarding where certain consent mechanisms must be displayed
-        """
-        consent_mechanism: Optional[str] = values.get("consent_mechanism")
-        displayed_in_overlay: Optional[bool] = values.get("displayed_in_overlay")
-        displayed_in_privacy_center: Optional[bool] = values.get(
-            "displayed_in_privacy_center"
-        )
```

Comment:
This is logic that was at the notice level but no longer makes sense.  I assume will likely need something similar at the experience level?